src/Transformer/AbstractObjectTransformer.php

```diff
@@ -9,13 +9,13 @@
 {
     abstract public function getObjectClass(): string;
 
-    public function toObject(ResourceInterface $resource)
+    public function toArray(ResourceInterface $resource)
     {
-        return $resource->untransform($this);
+        return $this->transform($resource);
     }
 
-    public function toArray(ResourceInterface $resource)
+    public function toObject(ResourceInterface $resource)
```

Comment:
what's the purpose of the `toObject` getter?
it seems misleads to "untransform" (even though objects do come it, transformers deal will arrayable data...

Reply:
I think he is untransforming to Object (like serialize/unserialize)

Reply:
ohh. because it's specifically on the `AbstractObjectTransformer`. makes sense